src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -285,11 +296,29 @@ private void AppendException(Exception currentException, IEnumerable<ExceptionRe
                 int beforeRenderLength = builder.Length;
                 var currentRenderFunction = _renderingfunctions[renderingFormat];
                 currentRenderFunction(builder, currentException);
+
                 if (builder.Length == beforeRenderLength && builder.Length != orgLength)
                 {
                     builder.Length = orgLength;
                 }
             }
+
+            if (_customRenderingfunctions.TryGetValue(currentException.GetType(), out var renderer))
+            {
+                try
+                {
+                    builder.Append(Separator);
+                    var render = renderer(currentException);
+                    builder.Append(render);
+                }
+                catch (Exception exception)
+                {
+                    var message = $"Exception in {typeof(ExceptionLayoutRenderer).FullName}.AppendException(): {exception.GetType().FullName}.";
+                    builder.Append("NLog message: ");
+                    builder.Append(message);
+                    InternalLogger.Warn(exception, message);
```

Comment:
Only log to internallog here?

Reply:
Probably, I copied it from `AppendMessage` in the same class

Reply:
Have to check this :)

Reply:
Only internal log please, thanks!

Reply:
@304NotModified Done